api/fncommon/fncommon.go

```diff
@@ -0,0 +1,26 @@
+package fncommon
```

Comment:
Why can't this be just a file inside API package? I see no reason for creating a package for 2 functions.

Reply:
This was requested by @treeder
https://fnproject.slack.com/archives/C7FKW83GV/p1511971979000041?thread_ts=1511947888.000097&cid=C7FKW83GV

Reply:
I kinda disagree with the placement. I see no point to define the whole package for 2 functions, I'm not seeing more fn parts be exposed outside except API extensions. So, all these changes eventually land in `api/common.go`.

Reply:
I'll leave you and @treeder to discuss this.

Reply:
@treeder, what’s the specific reason to define yet one more package for two function?

Reply:
Just to be clear, I don't have a strong view on this and am happy to go along with whatever @denismakogon  and @treeder want. 

For these two functions, I need to know:
* package name and * directory name. 

This PR (based on what I think @treeder asked for, to avoid possible name clashes in extensions that already have a `common` package) has
* package=`fncommon`
* directory= `api/fncommon`

I think @denismakogon is suggesting they stay where they are,:
* package=`common`
* directory= `api/common`

I suppose it partly depends on whether we intend to add additional API, for use by extensions, here in the future.

Reply:
We could throw these in fnext package that's part of this PR: https://github.com/fnproject/fn/pull/554 

Basically a package meant for extension developers to use.